src/css/components/_modal-dialog.scss

```diff
@@ -0,0 +1,34 @@
+.video-js .vjs-modal-dialog {
+  @extend %overlay-space;
+  @include linear-gradient(180deg, rgba(0, 0, 0, 0.8), rgba(255, 255, 255, 0));
+
+  .vjs-modal-dialog-content {
+    @extend %overlay-space;
+    z-index: 1;
+  }
+
+  // Ultimately, this would be moved to its own component partial, but
+  // for now it's unclear what styles would be generic and which specific
+  // to this context!
+  .vjs-close {
+    cursor: pointer;
+    font-size: 1.2em; // 12px
+    height: auto;
+    padding: 0.75em 1em; // 9px 12px
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: auto;
+    z-index: 2;
+
+    .vjs-control-text {
```

Comment:
Sure. If I had to hazard a guess, it's probably to avoid overly-specific selectors.